### PR TITLE
automate_virtual_uc_ctlpane_interface

### PR DESCRIPTION
**Pull request by Shrutikabansal**
automate  virtual_uc_ctlpane_interface for virtual_undercloud

### CODE REVIEW

**Comment by masco** [comment]
Can one of the admins verify this patch?

**Comment by Shrutikabansal** [comment]
@asyedham would you review it.

File: group_vars/all.yml [hunk 1]
```diff
@@ -153,7 +153,7 @@ virtual_uc: false
 undercloud_host: 172.16.0.2
 vm_external_interface: eth0
 undercloud_local_interface: eth0
-virtual_uc_ctlplane_interface: em1
+#virtual_uc_ctlplane_interface: em1
```
Comment (asyedham) [inline group_vars/all.yml]: remove this

**Review by asyedham — CHANGES_REQUESTED** [review]

File: group_vars/all.yml (hunk 1 above)
Comment (Shrutikabansal) [reply group_vars/all.yml]: @asyedham chanages done

**Pull request closed without merging**